frontend/src/component/insights/components/LineChart/ChartTooltip/ChartTooltip.tsx

```diff
@@ -91,12 +91,12 @@ export const ChartTooltipContainer: FC<IChartTooltipProps> = ({
     </Box>
 );
 
-export const ChartTooltip: VFC<IChartTooltipProps> = ({ tooltip }) => (
+export const ChartTooltip: FC<IChartTooltipProps> = ({ tooltip }) => (
     <ChartTooltipContainer tooltip={tooltip}>
         <Paper
             elevation={3}
             sx={(theme) => ({
-                width: 220,
+                width: 'max-content',
```

Comment:
Allows the tooltip to be grow to whatever size is needed for the content. Seems safe to me. The only other chart using this tooltip is flags, and I don't think that's in any danger of overflowing.

<img width="255" height="145" alt="Image" src="https://github.com/user-attachments/assets/a215d68a-3025-4ebf-a72a-768aeeb8ee47" />